src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSource.cs

```diff
@@ -529,13 +540,30 @@ private CodeRefactoring FilterOnUIThread(CodeRefactoring refactoring, Workspace
 
                         // diagnostic from things like build shouldn't reach here since we don't support LB for those diagnostics
                         Contract.Requires(diag.Item1.HasTextSpan);
-                        sets.Add(new SuggestedActionSet(group, priority, diag.Item1.TextSpan.ToSpan()));
+                        var category = GetFixCategory(diag.Item1.Severity);
+                        sets.Add(new SuggestedActionSet(category, actions: group, priority: priority, applicableToSpan: diag.Item1.TextSpan.ToSpan()));
                     }
                 }
 
                 return sets.ToImmutableAndFree();
             }
 
+            private static string GetFixCategory(DiagnosticSeverity severity)
+            {
+                switch (severity)
+                {
+                    case DiagnosticSeverity.Hidden:
+                    case DiagnosticSeverity.Info:
+                        return PredefinedSuggestedActionCategoryNames.Any;
+                    case DiagnosticSeverity.Warning:
+                        return PredefinedSuggestedActionCategoryNames.CodeFix;
+                    case DiagnosticSeverity.Error:
+                        return PredefinedSuggestedActionCategoryNames.ErrorFix;
```

Comment:
These are predefined lightbulb action categories. The default mappings are as follows:

- ErrorFix -> Error light bulb
- CodeFix -> default light bulb (CodeFix is based on the 'Any' category, which is mapped to the default bulb)
- Refactoring -> Screwdriver light bulb